modules/ROOT/pages/upgrade-cluster.adoc

```diff
@@ -145,7 +145,7 @@ cd /opt/anypoint/runtimefabric/installer
 . Run the command to resume the upgrade: 
 +
 ----
-sudo ./gravity upgrade --resume
+sudo gravity upgrade --resume
```

Comment:
same comment as above (which gravity binary to use in the case of upgrades)

Reply:
in this case the new version from the `installer` directory as that is the one running the upgrade